### DISCUSSION TITLE
Change access modifier for SpawnTableTool.resolve GUID override to public.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#4089] [Ideas]
With the spawntable editor supporting GUIDs, this should probably be made public.

I'm not sure if you already got to this when you fixed up the spawntable editor, because I'm assuming there will probably be a new method that returns a GUID rather than a ushort.

**Comment by SDGNelson** [comment]
There are actually quite a few public overloads in the next update for different use cases. 🙂

**Reply by SDGNelson** [reply]
![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/12567645/a1f41c2d-d199-4973-b42f-8b7bbc03a568)

**Reply by Jdance-Media** [reply]
Well then, I guess it is safe to close this. Naming conventions and plenty of overloads. Very excited. 👍
